wcivf/apps/elections/templates/elections/postcode_view.html

```diff
@@ -99,6 +99,36 @@
 
 
   </div>
+
+  {% if postelection.post.husting_set.count %}
+    <div class="card">
+      <h3>Hustings</h3>
+      {% for husting in postelection.post.husting_set.all %}
```

Comment:
I don't as a first pass, I'd rather have them in.
But quite possibly as a second iteration.